packages/core/addon/serializers/table.ts

```diff
@@ -164,6 +165,7 @@ export function normalizeTableV2(
   const columnAttributes = Object.values(columnData).reduce((columns, columnInfo) => {
     const { tableColumn, requestColumn } = columnInfo;
     const { attributes } = tableColumn;
+    assert(`The request column ${requestColumn.field} has a cid`, requestColumn.cid);
```

Comment:
```suggestion
    assert(`The request column ${requestColumn.field} does not have a present 'cid' field`, isPresent(questColumn.cid));
```

??

Reply:
I think we've used `should` pretty frequently for assert messages in the past. 
```suggestion
    assert(`The request column ${requestColumn.field} should have a present 'cid' field`, isPresent(questColumn.cid));
```